packages/validator/src/spectral/spectral-validator.js

```diff
@@ -161,24 +161,23 @@ function checkGetRulesetError(logger, error, chalk) {
     error.message.includes('ibm:oas')
   ) {
     logger.warn(
-      chalk.yellow('\n[Warning]') +
-        ' The IBM ruleset name has changed and the old one is invalid.\n' +
-        'Change your ruleset to extend `@ibm-cloud/openapi-ruleset` instead of `ibm:oas`\n' +
-        'to use your custom ruleset. For now, the IBM Spectral rules will run in their default configuration.'
+      'The IBM ruleset name has changed and the old name is invalid.'
+    );
+    logger.warn(
+      'Change your ruleset to extend `@ibm-cloud/openapi-ruleset` instead of `ibm:oas`'
+    );
+    logger.warn(
+      'to use your custom ruleset. For now, the IBM Spectral rules will run in their default configuration.'
     );
   } else if (
     error.message.endsWith('Cannot parse null because it is not a string')
   ) {
     logger.debug(
```

Comment:
Very picky 😅  but I think I like this as an "info" as originally scoped by the message. Feel free to change or leave

The one below makes more sense as a debug 👍 

```suggestion
    logger.info(
```

Reply:
I'll include this change in my next PR since I've got a bunch of changes in progress right now.